lib/service_check.pm

```diff
@@ -229,6 +229,9 @@ Check service before migration, zypper install service package, enable, start an
 =cut
 sub install_services {
     my ($service) = @_;
+    # trun off lmod shell debug information
+    script_run('echo export LMOD_SH_DBG_ON=1 >> /etc/bash.bashrc.local');
```

Comment:
using assert? i suppose this will always success?
same for below command

Reply:
OK